h2o-py/tests/testdir_algos/automl/pyunit_automl_args.py

```diff
@@ -298,10 +298,55 @@ def test_frames_cannot_be_passed_as_key():
             assert "'{}' must be a valid H2OFrame".format(attr) in str(e)
 
 
+def test_no_time_limit_if_max_models_is_provided():
+    ds = import_dataset()
+    aml = H2OAutoML(project_name="py_no_time_limit", seed=1, max_models=1)
+    aml.train(y=ds['target'], training_frame=ds['train'])
+    max_runtime = aml._build_resp['build_control']['stopping_criteria']['max_runtime_secs']
+    max_models = aml._build_resp['build_control']['stopping_criteria']['max_models']
+    assert max_runtime == 0
+    assert max_models == 1
+
+
+def test_max_runtime_secs_alone():
+    ds = import_dataset()
+    aml = H2OAutoML(project_name="py_max_runtime_secs", seed=1, max_runtime_secs=7)
+    aml.train(y=ds['target'], training_frame=ds['train'])
+    max_runtime = aml._build_resp['build_control']['stopping_criteria']['max_runtime_secs']
+    max_models = aml._build_resp['build_control']['stopping_criteria']['max_models']
+    assert max_runtime == 7
+    assert max_models == 0
+
+
+def test_max_runtime_secs_can_be_set_in_combination_with_max_models():
+    ds = import_dataset()
+    aml = H2OAutoML(project_name="py_all_stopping_constraints", seed=1, max_models=1, max_runtime_secs=12)
+    aml.train(y=ds['target'], training_frame=ds['train'])
+    max_runtime = aml._build_resp['build_control']['stopping_criteria']['max_runtime_secs']
+    max_models = aml._build_resp['build_control']['stopping_criteria']['max_models']
+    assert max_runtime == 12
```

Comment:
@seb-h2o can we test that actually max_runtime_secs is being ignored?

Reply:
why ignored? it's not ignored, it's the first that wins.
I thought we already had a test though checking this scenario in `pyunit_automl_args.py` when both are set, but apparently not: will add.

Reply:
@seb-h2o 
> why ignored? it's not ignored, it's the first that wins.

you are right. forgot for a second